include/profiler.h

```diff
@@ -19,14 +19,21 @@
 #include <sys/util.h>
 #include <sys/__assert.h>
 
-#ifndef CONFIG_PROFILER_MAX_NUMBER_OF_CUSTOM_EVENTS
-/** Maximum number of custom events. */
-#define CONFIG_PROFILER_MAX_NUMBER_OF_CUSTOM_EVENTS 0
+#ifndef CONFIG_PROFILER_MAX_NUMBER_OF_EVENTS
+/** Maximum number of custom events. Note that there are two hidden events for tracking event
```

Comment:
> Note that there are two hidden events for tracking event processing start and stop in Event Manager

This part does not belong to profiler.

Reply:
Ok, I agree that info about those two "hidden" events should be primely placed in Event Managers documentation, but I think it is also a good place to note it. This config can be wrongly set by user if he doesn't know of this fact. I think we should also add this remark to EVENT_MANAGER_MAX_EVENT_CNT.

Reply:
Maybe it should be moved to Kconfig help?